OCR/tests/assets/labels.json

```diff
@@ -1,4 +1,10 @@
-{
-  "255,0,0": "nbs_patient_id",
-  "0,0,255": "nbs_cas_id"
-}
\ No newline at end of file
+[
+  {
+    "color": "255,0,0",
+    "label": "nbs_patient_id"
```

Comment:
Should these have a `type` field as well?

Reply:
Good Catch! though unused right now, it should be there

Reply:
Fixed, thanks